text_summary.py

```diff
@@ -0,0 +1,30 @@
+import torch
+from transformers import T5Tokenizer, T5ForConditionalGeneration
+
+model = T5ForConditionalGeneration.from_pretrained('t5-small') # use pretrained model
+tokenizer = T5Tokenizer.from_pretrained('t5-small') # use pretrained tokenizer
+device = torch.device('cpu')
+
+
+def text_summary(text):
```

Comment:
We have created a .txt file with the entire captions text inside `videos/<name_of_video>/script.txt`. Look at #11. In the file, I have commented where you need to call your model and where the script file is. So maybe make a function, read the text from that file and then pass it on to the model.

Reply:
Will wait for merges to actualize this. The function will just be a read function. It is independent of my summary function. So I will be implementing it in the clipbit.py file